best_practices/controllers.rst

```diff
@@ -209,8 +209,7 @@ flexible::
     }
 
 The point is this: the ParamConverter shortcut is great for simple situations.
-But you shouldn't forget that querying for entities directly is still very
-easy.
+But you shouldn't forget that querying for entities directly is not to difficult.
```

Comment:
What about:

> You can still query entities directly outside of parameter converters.

Or something like that. The whole sentence is off imo. If you want to keep it the way it is, change `to` to `too`

Reply:
"But you shouldn't forget that querying for entities directly is still possible" ?

Reply:
It's the "but" part that throws it off imo

Reply:
maybe "However, there is nothing wrong with querying for entities directly if the ParamConverter would get complicated."

Reply:
I like your proposal @dbu 👍